doc/user story - tracking.md

```diff
@@ -0,0 +1,11 @@
+Here is the simple user story for how a movie is uploaded and tracked.
+
+1. User uploads a movie with the **upload** feature. Movie is uploaded as an MP4.
+2. User click "analyze." Screen clears and shows the first frame of the movie with the tools for adding and moving trackpoints.
+3. User identifies four points: ruler0 and ruler10 (the start and stop points on the ruler), plant1 and plant2 (the two control points on the plant)
+4. User clicks "track movie"
+5. The JavaScript in analyze.js sends the trackpoints to the server. On the server a loop runs which tracks the points on each frame and producing a new movie.
+6. The new movie is stored as a derivative movie. The track points are added as frame-by-frame annotations to the new movie, but we don't store all of the frames (we have the medata for each frame)
+7. The JavaScript shows the tracked movie, which can be played. (Right now we won't handle what happens if tracking is lost.)
```

Comment:
That's fine for the happy path use case, but the sad path of tracking loss mid-movie case is one that we will run across pretty much immediately.

Reply:
Well, I want to know the API calls for the happy path. I think that we can:
- number every frame.
- If we loose tracking, we start the analysis at frame X. 
- So we need an API call to extract frame X (pretty easy to do, in fact; perhaps we do not need to store every frame.)
- But my goal is to get the happy path working first.
-